src/pybind/mgr/dashboard/tox.ini

```diff
@@ -140,3 +142,27 @@ commands =
 [testenv:check]
 commands =
     python ci/check_grafana_uids.py frontend/src/app ../../../../monitoring/grafana/dashboards
+
+[testenv:openapi-{check,fix}]
```

Comment:
As discussed, worth considering whether or not a diff should trigger the `testenv:openapi-doc` protocol. However, that may be too much and manual updating of the documents could be preferable.

Reply:
Understood. This process is still consistent with other checks (like linting), where changes are blocked if linting fails and an auto-fixer exists, but the user yet needs to manually fix linting errors and resubmit changes. I'm not sure if we can/should modify code automatically at post-commit stage... (if that's what you're suggesting).